build/webpack/webpack.config.base.js

```diff
@@ -20,6 +22,49 @@ class AccessDependenciesPlugin {
   }
 }
 
+const defines = {
+  BUILDFLAG: ' '
+}
+
+if (buildFlagArg) {
+  const buildFlagPath = buildFlagArg.substr(13)
+
+  const flagFile = fs.readFileSync(buildFlagPath, 'utf8')
+  for (const line of flagFile.split(/(\r\n|\r|\n)/g)) {
+    const flagMatch = line.match(/#define BUILDFLAG_INTERNAL_(.+?)\(\) \(([01])\)/)
+    if (flagMatch) {
+      defines[flagMatch[1]] = JSON.stringify(Boolean(parseInt(flagMatch[2], 10)))
```

Comment:
Maybe give name to matches? Otherwise it's not clear what's going on here.
```js
const flagName = flagMatch[1];
const flagValue = flagMatch[2];
```